plugins/foreman_discovery/4.0/index.md

```diff
@@ -707,6 +695,59 @@ foreman-discovery-image git
 [repository](https://github.com/theforeman/foreman-discovery-image) and find
 the README for further instructions.
 
+## 5.3 PXE-less discovery
+
+It is possible to use Discovery image directly as a CDROM/DVDROM ISO that can
+be also transferred to hard drive or USB stick (dd command). In this case,
+automatic discovery process is not started and user select primary network
+interface interactively. On the couple of screens user configures network
+credentials (IPv4 supported at the moment), Foreman URL and type and fills user
+facts which are then uploaded to the server.
```

Comment:
Some minor corrections:

It is possible to use **the** Discovery image directly as a CDROM/DVDROM ISO that can
be also transferred to hard drive or USB stick (dd command). In this case,
automatic discovery process is not started and **the** user **can** select **the** primary network
interface interactively. On the **next** couple of screens**, the** user configures network
credentials (IPv4 supported at the moment), **the** ~~Foreman~~ URL and type **(Foreman or Smart Proxy)**, and fills **custom** user facts which are then uploaded to the server.